third_party/remoteapis/bazel_remote_apis.patch

```diff
@@ -0,0 +1,80 @@
+diff --git a/remote_apis_deps.bzl b/remote_apis_deps.bzl
+index 94290e3..dabe4da 100644
+--- a/remote_apis_deps.bzl
++++ b/remote_apis_deps.bzl
+@@ -1,7 +1,7 @@
+ """Load dependencies needed to depend on the RE API repo."""
+ 
+-load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies", "go_repository")
+-load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
++# load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies", "go_repository")
++# load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
```

Comment:
I think it'd be better if we just remove the lines instead of commenting them out (below, too) - the patch is easier to read that way.